testdata/p4_16_samples/table-entries-exact-bmv2.p4

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
there are default entry tests in almost all the test cases. Again, the way I expect defaults for tuples is ```(_, _)```.